infra/production/base/module/ecs/main.tf

```diff
@@ -0,0 +1,44 @@
+resource "aws_ecs_cluster" "cluster" {
```

Comment:
TLDR: This file defines the ECS cluster, IP Whitelist for ECS Instances, and specifies where the ECS can obtain an EC2 instance to run the container.